projects/packages/changelogger/src/Plugins/WordpressVersioning.php

```diff
@@ -47,7 +47,7 @@ public function getOptions() {
 	 * @throws InvalidArgumentException If the version number is not in a recognized format.
 	 */
 	private function parseVersion( $version ) {
-		if ( ! preg_match( '/^(?P<major>\d+\.\d)(?:\.(?P<point>\d+))?(?:-(?P<prerelease>dev|(?:alpha|beta|rc)\d*))?(?:\+(?P<buildinfo>[0-9a-zA-Z.-]+))?$/', $version, $m ) ) {
+		if ( ! preg_match( '/^(?P<major>\d+\.\d)(?:\.(?P<point>\d+))?(?:-(?P<prerelease>dev|(?:alpha|beta|rc|[0-9]*)\d*))?(?:\+(?P<buildinfo>[0-9a-zA-Z.-]+))?$/', $version, $m ) ) {
```

Comment:
That regex probably won't work well, both because it'll accept like "10.1-" and the "[0-9]*\d*" may be inefficient.

Let's go back to what you originally were looking at that was like "10.1-YYMM.n"
```suggestion
		if ( ! preg_match( '/^(?P<major>\d+\.\d)(?:\.(?P<point>\d+))?(?:-(?P<prerelease>dev|(?:alpha|beta|rc)\d*|\d\d(?:0[1-9]|1[0-2])\.\d+))?(?:\+(?P<buildinfo>[0-9a-zA-Z.-]+))?$/', $version, $m ) ) {
```